.travis.yml

```diff
@@ -32,14 +36,12 @@ before_install:
         popd;
     fi
 
-
 # all commands here must have exit code 0 for the build to be called "passing"
 # debugging: octave -W --no-gui --eval "syms x; A = [x sin(x) x^3]; A; exit(0)"
 script:
   - octave -W --no-gui --eval "ver; pwd; exit(0)"
-  - stty cols 80 rows 40
+  - stty cols $COLUMNS rows 40
```

Comment:
Hmmm, I had an actual problem with this.  I can try to track it down, but I'm scared of this change b/c of some travis bug IIRC.